src/mammos_entity/io.py

```diff
@@ -274,25 +288,29 @@ def _entities_to_csv(
     **entities: mammos_entity.Entity | astropy.units.Quantity | numpy.typing.ArrayLike,
```

Comment:
I would change the signature of all writing functions, rename `_description` -> `description` and allow it as a positional or keyword argument.